Makefile

```diff
@@ -2,91 +2,67 @@ SHELL = /usr/bin/env bash
 OS_NAME = $(shell uname -s)
 BUILD_CONFIGURATION = Debug
 BINARIES_PATH = $(shell pwd)/Binaries
-SCRIPTS_PATH = $(shell pwd)/build/scripts
 SRC_PATH = $(shell pwd)/src
-TOOLSET_SRC_PATH = $(shell pwd)/build/MSBuildToolset
-TOOLSET_PATH = $(BINARIES_PATH)/toolset
-RESTORE_SEMAPHORE_PATH = $(BINARIES_PATH)/restore.semaphore
 BOOTSTRAP_PATH = $(BINARIES_PATH)/Bootstrap
 BUILD_LOG_PATH =
 HOME_DIR = $(shell cd ~ && pwd)
-DOTNET_VERSION = 1.0.0-preview3-003223
-NUGET_VERSION = 3.5.0-beta2
-NUGET_EXE = $(shell pwd)/nuget.exe
+DOTNET_VERSION = 1.0.1
+DOTNET = $(BINARIES_PATH)/dotnet-cli/dotnet
+TARGET_FX = netcoreapp1.1
 
 MSBUILD_ADDITIONALARGS := /v:m /fl /fileloggerparameters:Verbosity=normal /p:Configuration=$(BUILD_CONFIGURATION)
 
+
 ifeq ($(OS_NAME),Linux)
-	MSBUILD_ADDITIONALARGS := $(MSBUILD_ADDITIONALARGS) /p:BaseNuGetRuntimeIdentifier=$(shell . /etc/os-release && echo $$ID.$$VERSION_ID)
+	RUNTIME_ID  := $(shell . /etc/os-release && echo $$ID.$$VERSION_ID)-x64
```

Comment:
Why are there two spaces between `RUNTIME_ID` and `:=`?  Can't tell if this is necessary or accident.  Make file spacing is important but don't think for this case.  This pattern is repeated a couple times in this file.